src/Administration/Resources/app/administration/src/app/init-pre/state.init.ts

```diff
@@ -32,40 +32,12 @@ function registerProperties(state: FullState, store: Store<VuexRootState>) {
     /* eslint-disable @typescript-eslint/no-unsafe-return, @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-argument, max-len */
     state._registerPrivateProperty('_store', store);
     state._registerProperty('list', () => Object.keys(store.state));
-    state._registerProperty('get', <N extends keyof VuexRootState | 'cmsPageState' | 'cmsPage'>(name: N) => {
-        /* @deprecated: tag:v6.7.0 - Will be removed without replacement */
-        if (name === 'cmsPageState' || name === 'cmsPage') {
-            Shopware.Utils.debug.warn(
-                'Core',
-                'Shopware.State.get("cmsPageState") is deprecated! Use Shopware.Store.get("cmsPage") instead.',
-            );
-
-            return Shopware.Store.get('cmsPage');
-        }
-
+    state._registerProperty('get', <N extends keyof VuexRootState | 'cmsPage'>(name: N) => {
```

Comment:
please double check this as well, i was not sure what the intention of the deprecation was as well

Reply:
@tajespasarela

Reply:
This can be removed as the `cmsPageState` store no longer exists, and there is only the `cmsPage` which is already migrated to Pinia.

So this change is ok.